src/miragejs/server.js

```diff
@@ -0,0 +1,41 @@
+import { Server, Model, Serializer } from "miragejs";
+import { routes } from "./routes/index.js";
+var db = require("./db.json");
+
+const CustomSerializer = Serializer.extend({
+  serialize(object) {
+      const json = Serializer.prototype.serialize.apply(this, arguments);
+      const { modelName } = object;
+      return json[modelName];
+  }
+})
+
+export function makeServer({ environment = "development" } = {}) {
+  let server = new Server({
+    environment,
+
+    serializers: {
+      application: CustomSerializer
+    },
+
+    models: {
+      node: Model,
+      obm: Model,
+      poller: Model,
+      profile: Model,
+      sku: Model,
+      template: Model,
+      graph: Model,
+      catalog: Model,
+      config: Model,
+      file: Model,
+      workflow: Model
+    }
+  });
+
+  server.routes = routes(server);
```

Comment:
you can also pass the `routes` as a parameter to the `new Server`, but this works so 👍 .

If the `baseURL` and the `namespace` are always the same, I'd pass them as a parameter to the `new Server` too. It makes it a little bit clearer

Reply:
Yeap, nice! I did the same with the data base, I think it also looks better inside the server